yarn-project/end-to-end/src/e2e_public_token_contract.test.ts

```diff
@@ -72,7 +75,13 @@ describe('e2e_public_token_contract', () => {
     const receipt = await tx.getReceipt();
 
     expect(receipt.status).toBe(TxStatus.MINED);
-    await expectAztecStorageSlot(logger, aztecRpcServer, contract, balanceSlot, recipient.toField(), mintAmount);
+
+    const balance = await cc.l2.loadPublic(
```

Comment:
I prefered the shortness of the previous diff, i agree with rahul a helper function can be used here